src/content/chainlink-functions/resources/simulation.mdx

```diff
@@ -0,0 +1,78 @@
+---
+section: chainlinkFunctions
+date: Last Modified
+title: "Simulate your Functions"
+---
+
+import { ClickToZoom, Aside } from "@components"
+
+Before making a Chainlink Functions request from your smart contract, it is always a good practice to simulate the source code off-chain to make any adjustments or corrections.
+
+Currently, there are several options for simulating a request:
+
+- [Chainlink Functions playground](https://functions.chain.link/playground).
+
+- [Chainlink Functions Hardhat Starter Kit](https://github.com/smartcontractkit/functions-hardhat-starter-kit): Use the `npx hardhat functions-simulate-script` command to simulate your Functions JavaScript source code.
+
+- [Chainlink Functions Toolkit NPM package](https://github.com/smartcontractkit/functions-toolkit): Import this NPM package into your JavaScript/TypeScript project, then use the `simulateScript` function to simulate your Functions JavaScript source code.
+
+## Chainlink Functions playground
+
+<Aside type="caution">
+  The simulation is run on the browser. There may be differences to the real Chainlink Functions environment. For
+  instance, you are limited to vanilla JavaScript and cannot use all the functionalities of Chainlink Functions, such as
+  Imports of external modules. To access all the functionalities, use the [Chainlink Functions Hardhat Starter
+  Kit](https://github.com/smartcontractkit/functions-hardhat-starter-kit) or [Chainlink Functions Toolkit NPM
+  package](https://github.com/smartcontractkit/functions-toolkit).
+</Aside>
+
+To explore the capabilities of Chainlink Functions, visit the [Chainlink Functions Playground](https://functions.chain.link/playground). This playground can simulate Chainlink Functions within the browser, call APIs, and execute demo requests.
```

Comment:
could you combine this paragraph with the disclaimer text you put above to streamline the disclaimer text? start with this sentence and then put the limitations after it?